server.go

```diff
@@ -151,6 +151,12 @@ func main() {
 		log.WithField("statsdUrl", s.StatsdUrl).Fatal("Could not connect to statsd")
 	}
 
+	if v := os.Getenv("EXCLUDED_TAGS"); v != "" {
+		for _, t := range strings.Split(v, ",") {
```

Comment:
I hesitated with a regexp instead. But the tags that _can_ be used are whitelisted in the first place ([here](https://github.com/deliveroo/heroku-datadog-drain-golang/blob/master/client.go#L21-L24) so there's not much of a point - at most you can exclude that list.